dev-start.sh

```diff
@@ -104,12 +104,30 @@ changeNodeVersion() {
   fi
 }
 
+setupLocalKinesis() {
+  sleep 3s
+  # java sdk use CBOR protocol
+  # which does not work with localstack kinesis which use kinesislite
+  export AWS_CBOR_DISABLE=true
+  # dummy env variables for local kinesis
+  export AWS_ACCESS_KEY_ID=foobar
+  export AWS_SECRET_ACCESS_KEY=foobar
+  export AWS_REGION=eu-west-2
```

Comment:
why a different region from the rest of the app (eu-west-1)?

Reply:
it does not matter it is localhost just for AWS cli this variable need to exists with anything, will change to the same for consistency

Reply:
done